packages/airnode-protocol/test/rrp/requesters/RrpBeaconServerV0.sol.js

```diff
@@ -29,50 +28,92 @@ describe('RrpBeaconServerWithManager', () => {
     };
     const accessControlRegistryFactory = await hre.ethers.getContractFactory('AccessControlRegistry', roles.deployer);
     accessControlRegistry = await accessControlRegistryFactory.deploy();
-    const airnodeRrpFactory = await hre.ethers.getContractFactory('AirnodeRrp', roles.deployer);
+    const airnodeRrpFactory = await hre.ethers.getContractFactory('AirnodeRrpV0', roles.deployer);
     airnodeRrp = await airnodeRrpFactory.deploy();
-    const rrpBeaconServerFactory = await hre.ethers.getContractFactory('RrpBeaconServer', roles.deployer);
+    const rrpBeaconServerFactory = await hre.ethers.getContractFactory('RrpBeaconServerV0', roles.deployer);
     rrpBeaconServer = await rrpBeaconServerFactory.deploy(
       accessControlRegistry.address,
       rrpBeaconServerAdminRoleDescription,
       roles.manager.address,
       airnodeRrp.address
     );
     const managerRootRole = await accessControlRegistry.deriveRootRole(roles.manager.address);
+    // Initialize the roles and grant them to respective accounts
```

Comment:
This stuff can be done with a single tx because AccessControlRegistry now inherits Multicall